@Barry-Xu-2018 i do not quite understand why these CI failures (except windows 😓 ) are happening, do you have any thoughts?

```console
Traceback (most recent call last):
  File "/home/jenkins-agent/workspace/ci_linux-aarch64/ws/src/ros2/ros2cli/ros2action/test/test_echo.py", line 281, in test_echo
    assert action_command.wait_for_output(
AssertionError: assert <BoolWithText(False): 'Failed to load entry point \'echo\': No module named \'ros2topic\'\nTraceback (most recent call...s_on_demand\n    extension = command_extensions[name]\n                ~~~~~~~~~~~~~~~~~~^^^^^^\nKeyError: \'echo\'\n'>
 +  where <BoolWithText(False): 'Failed to load entry point \'echo\': No module named \'ros2topic\'\nTraceback (most recent call...s_on_demand\n    extension = command_extensions[name]\n                ~~~~~~~~~~~~~~~~~~^^^^^^\nKeyError: \'echo\'\n'> = <bound method ProcessProxy.wait_for_output of <launch_testing.tools.process.ProcessProxy object at 0xea9cc9371f40>>(functools.partial(<function match_expected_output at 0xea9cd1286520>, expected_output=[['interface: GOAL_SERVICE', 'in...[1], length: 0>'", "response: '<sequence type: test_msgs/action/Fibonacci_GetResult_Response[1], length: 1>'", '---']]), timeout=10)
 +    where <bound method ProcessProxy.wait_for_output of <launch_testing.tools.process.ProcessProxy object at 0xea9cc9371f40>> = <launch_testing.tools.process.ProcessProxy object at 0xea9cc9371f40>.wait_for_output
 +    and   functools.partial(<function match_expected_output at 0xea9cd1286520>, expected_output=[['interface: GOAL_SERVICE', 'in...[1], length: 0>'", "response: '<sequence type: test_msgs/action/Fibonacci_GetResult_Response[1], length: 1>'", '---']]) = <class 'functools.partial'>(match_expected_output, expected_output=[['interface: GOAL_SERVICE', 'info:', '  event_type: REQUEST_SENT', '  stamp:', re.compile('    sec: \\d+'), re.compil...id:', "  uuid: '<array type: uint8[16]>'", 'feedback:', "  sequence: '<sequence type: int32, length: 3>'", '---'], ...])
 +      where <class 'functools.partial'> = functools.partial
```

--- a/ros2action/package.xml
+++ b/ros2action/package.xml
@@ -21,6 +21,7 @@
   <exec_depend>ament_index_python</exec_depend>
   <exec_depend>rclpy</exec_depend>
   <exec_depend>ros2cli</exec_depend>
+  <exec_depend>ros2topic</exec_depend>
   <exec_depend>rosidl_runtime_py</exec_depend>
 
   <test_depend>ament_copyright</test_depend>

